PEP 467: Defer, and update open questions

ncoghlan opened the pull request:
<!--

Please include the PEP number in the pull request title, example:

PEP NNN: Summary of the changes made

In addition, please sign the CLA.

For more information, please read our Contributing Guidelines (CONTRIBUTING.rst)

-->

ncoghlan commented:
@ethanfurman I ended up make some other edits while deferring the PEP, so it would be good if you could take a look at those before I hit merge :)

gvanrossum commented on pep-0467.txt:
```diff
@@ -73,8 +62,30 @@ second argument is the fill byte to use (defaults to ``\x00``)::
+single integer, while allowing for non-zero fill values when needed.
+
+
+Deprecation of current "zero-initialised sequence" behaviour without removal
```
IMO that should not be called "deprecation". "Discouragement" sounds right though.

ethanfurman replied:
I'm fine with "discouragement".

ncoghlan replied:
Done, the word "deprecate" and variants thereof no longer appears in the PEP.

ncoghlan commented:
Updated to avoid any mention of deprecation. While doing that, I also reverted one of my other changes, and put the sections back in their original order (the `fromsize` method descriptions make more sense after the PEP has explained the existing constructor behaviour).

And then while rearranging those sections I noticed that the examples implied that the parameters were positional-only, but didn't outright say that.
